test/heartbeat/heartbeat.js

```diff
@@ -43,42 +45,45 @@ const generator1Updated = {
   aspects: [{ name: 'A1', timeout: '1m' }],
   generatorTemplateName: 'refocus-trust1-collector',
   subjectQuery: 'absolutePath=Parent.Child.*&tags=Primary',
-  context: { baseUrl: 'https://example.api', },
+  context: { baseTrustUrl: 'https://example.api', },
   collectors: [{ name: 'agent1' }],
-  generatorTemplate: {
-    name: 'refocus-trust1-collector',
-    connection: {
-      url: 'https://example.api/v2',
-      bulk: true,
-    },
+  generatorTemplate: sgt,
+  refocus: {
+    url: refocusUrl,
   },
+  token: 'mygeneratorusertoken',
   interval: 12000,
 };
 
 const generator2 = {
   name: 'generator2_heartbeat',
   interval: 6000,
   aspects: [{ name: 'A2', timeout: '1m' }],
+  subjectQuery: '?absolutePath=S1.S2',
   subjects: [{ absolutePath: 'S1.S2', name: 'S2' }],
```

Comment:
We dont need the "subjects" attribute here anymore, correct?

Reply:
We don't, but can we tackle cleaning that up when we handle the "bulk=false" story?